docs/guides/getting-started/setup/integrate.mdx

```diff
@@ -0,0 +1,141 @@
+---
+description: If you already have an existing web project, this guide will walk you through integrating Tauri into your project
+pagination_next: guides/development/development-cycle
+sidebar_position: 3
+sidebar_custom_props:
+  doc_card_image: /img/guides/getting-started/setup/integrate-light.svg
+  doc_card_image_dark: /img/guides/getting-started/setup/integrate-dark.svg
+---
+
+import Commands from './_fragments/_commands.mdx'
+import TauriInit from './_fragments/_tauri-init.mdx'
+import Command, { InstallTauriCli, InstallTauriApi } from '@theme/Command'
+
+# Integrate into Existing Project
+
+If you already have an existing web project, this guide will walk you through integrating Tauri into your project, whether it is Node.js-based (like [Svelte], [React], [Vue], or [Angular]) or Rust-based (like [Yew] or [Dominator]).
+
+:::info
+
+Before we continue, make sure you have completed the [prerequisites] to have a working development environment.
+
+:::
+
+Although Tauri is compatiable with nearly any frontend framework, we'll use a [React] project throughout this guide created using [create-react-app]. We'll be assuming you're starting off with a project structure similar to this:
+
+```
+.
+│── package.json
+│── public
+│   ╰── index.html
+╰── src
+    │── App.css
+    │── App.jsx
+    │── index.css
+    ╰── index.js
+```
+
+## Create the Rust Project
+
+<TauriInit
+  destDirExplination={{
+    __html:
+      "If using our React example this is <code>../build</code>. Note that it may be something different like <code>../dist</code> if you're using a different framework.",
```

Comment:
```suggestion
      "For the project example in this guide , this is <code>../build</code>. Note that it may be something different like <code>../dist</code> if you're using a different framework.",
```